docs/reference/commandline/build.md

```diff
@@ -79,7 +79,11 @@ Build Syntax Suffix | Commit Used | Build Context Used
 Instead of specifying a context, you can pass a single Dockerfile in the `URL`
 or pipe the file in via `STDIN`. To pipe a Dockerfile from `STDIN`:
 
-    docker build - < Dockerfile
+    $ docker build - < Dockerfile
+
+With Powershell on Windows, you can run:
+
+    Get-Content Dockerfile | docker build -
```

Comment:
Haven't tested. It's worth debating whether we should have examples in `cmd.exe` too. I don't know what's the usage of different shells on Windows, but I somehow hope most people are on Powershell (if they're not using a Windows Bash implementation)